java/code/src/com/suse/manager/webui/controllers/contentmanagement/ContentManagementViewsController.java

```diff
@@ -174,6 +174,20 @@ public static ModelAndView listFiltersView(Request req, Response res, User user)
         if (req.queryParams("projectLabel") != null) {
             data.put("projectLabel", req.queryParams("projectLabel"));
         }
+        String openTemplate = req.queryParams("openTemplate");
+        if (openTemplate != null) {
+            data.put("openTemplate", openTemplate);
+
+            String systemId = req.queryParams("systemId");
+            String systemName = req.queryParams("systemName");
+            String kernelName = req.queryParams("kernelName");
```

Comment:
After the latest discussion, I guess `kernelName` can be removed from the whole chain

Reply:
`kernelName` is used in the system display name in the dropdown, but this part became redundant regardless thanks to moving the URL params bit to the frontend.